### PR TITLE
target: tizenrt-artik053: Update porting

### PR DESCRIPTION
**Pull request by hs0225**
- rename artik05x to artik053
- fix to add library for jerryscript
- add rom patch for artik053
- enable jerry cmd in tizenrt
- change to absolute file path when loading file with jerry cmd
- update README.md and config file

JerryScript-DCO-1.0-Signed-off-by: Hosung Kim hs852.kim@samsung.com

### CODE REVIEW

**Comment by glistening** [comment]
@hs0225 romfs.patch seems missing.

**Comment by hs0225** [comment]
@glistening I added a patch file. Thank you.

File: targets/tizenrt-artik053/apps/jerryscript/jerry_main.c [hunk 1]
```diff
@@ -312,17 +313,13 @@ register_js_function (const char *name_p, /**< name of the function */
...
+ * JerryScript command main
  */
-#ifdef CONFIG_BUILD_KERNEL
-int main (int argc, FAR char *argv[])
-#else
-int jerry_main (int argc, char *argv[])
-#endif
-{
+static int
+jerry_cmd_main (int argc, char *argv[]) {
```
Comment (LaszloLango) [inline targets/tizenrt-artik053/apps/jerryscript/jerry_main.c]: `{` should be in the next line.

**Comment by hs0225** [comment]
@LaszloLango I updated as your commented. Thank you for review.

**Comment by yichoi** [comment]
LGTM

**Pull request merged**